examples/java/src/main/java/com/xebia/functional/xef/java/auto/ASCIIArt.java

```diff
@@ -1,14 +1,17 @@
 package com.xebia.functional.xef.java.auto;
 
+import java.util.concurrent.ExecutionException;
+
 public class ASCIIArt {
     public static class Art {
         public String art;
```

Comment:
It’s confusing to have a class member with the same name (case differences aside) as its enclosing class. This is particularly so when you consider the common practice of naming a class instance for the class itself.
Best practice dictates that any field or member with the same name as the enclosing class be renamed to be more descriptive of the particular aspect of the class it represents or holds.

Noncompliant Code Example

public class Foo {
  private String foo;

  public String getFoo() { }
}

Foo foo = new Foo();
foo.getFoo() // what does this return?

you can rename the field art or remove the class Art and let the field art from the class ASCIIArt